Modules/Microsoft365DSC/Modules/M365DSCUtil.psm1

```diff
@@ -3174,7 +3174,12 @@ function Get-M365DSCExportContentForResource
 
         [Parameter()]
         [System.Management.Automation.PSCredential]
-        $Credential
+        $Credential,
+
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
```

Comment:
This will throw errors since the current calls made to the function never pass in TenantId. We would need to modify the logic for almost every resource if we validate notnullorempty here.

Reply:
Fair enough, I removed that validation.